lib/order.rb

```diff
@@ -0,0 +1,55 @@
+class Order 
+  attr_reader :my_order 
+
+  def initialize
+    @my_order = []
+  end
+
+  def add_dish_to_order(name, quantity)
+    @name, @quantity = name, quantity
+    @my_order << { @name => @quantity }
+    return "#{quantity} x #{name} added to your order"
+  end
+
+  def view_order
+    order_display
+  end
+
+  def order_total
+    sum = 0
+    menu = Menu.new.access_menu 
```

Comment:
We have a dependency on `Menu` in this class, so when testing, we'd ideally want to isolate our `Order` class from it. This can be done if we dependency-inject the `Menu` instance into Order, rather than creating it here - it would look like this:
```
class Order
  def initialize(menu = Menu.new)
     @menu = menu
    # ...
```

Then you could use a double in your tests to pass it instead of the "real" Menu instance, in order to mock its behaviour